pkgs/games/dwarf-fortress/wrapper/default.nix

```diff
@@ -58,6 +59,10 @@ let
   '' + lib.optionalString enableTWBT ''
     substituteInPlace $out/data/init/init.txt \
       --replace '[PRINT_MODE:2D]' '[PRINT_MODE:TWBT]'
+  '' + 
+ lib.optionalString enableTextMode ''
```

Comment:
I feel like it would be more readable to merge these two lines (62 and 63). I would have it read something like:
`'' + lib.optionalString enableTextMode ''`.

Reply:
Will do that if I add or edit anything again. Text mode was a bit important for me as I can SSH into it from my phone now.
"Pffft. Linux, you say? Grow a beard and enable the ncurses console mode, you GUI ne'er-do-well.

```cp init.txt backup.txt && sed 's/PRINT_MODE:.*\]/PRINT_MODE:TEXT\]/' backup.txt > init.txt```

I tell you true: mortal jaws slack — and maidens' underthings do drop — to see a proper /dev/tty1 fortress, fully realized and wrapped in visual mystery. Stonesense... that will earn you naught but the scorn of men.
"
(Warning, this doesn't work on NixOS as init.txt is read only)